linux_cpu_usage.py

```diff
@@ -47,33 +92,16 @@ def get_cpustats_dict(header):
             for i in header:
                 cpustats_dict[name][i] = x.pop(0)
         elif line.startswith('btime '):
-            uptime = int(time.time()) - int(line.split(' ',1)[1])
+            uptime = int(time()) - int(line.split(' ', 1)[1])
         elif line.startswith('intr '):
-            interrupts = int(line.split(' ',2)[1])
+            interrupts = int(line.split(' ', 2)[1])
         elif line.startswith('ctxt '):
-            context_switches = int(line.split(' ',1)[1])
-
-    return(cpustats_dict, total_dict, uptime, count, interrupts, context_switches)
-
-
-now = str(int(time.time()))
-graphite_data=''
-hostname = socket.gethostname().replace('.','_')
-sector_size=512
-
-header=['user','nice','system','idle','iowait','irq','softirq','steal']
-cs, totals, uptime, count, intr, ctxt = get_cpustats_dict(header)
-
-for cpu in cs:
-    for metric in header:
-        graphite_data += 'servers.%s.system.cpu.%s.%s %s %s\n' % (hostname,cpu, metric, str(cs[cpu][metric]), now )
+            context_switches = int(line.split(' ', 1)[1])
 
-for value in totals:
-    graphite_data += 'servers.%s.system.cpu.%s %s %s\n' % (hostname, str(value), totals[value], now )
+    return (
+        cpustats_dict, total_dict, uptime, count, interrupts, context_switches
```

Comment:
IDE says, that `interrupts` and `context_switches` might be not assigned here. Maybe you need some default assignments somewhere.